src/templates/can-observable-object/observable-object-static-properties-output.js

```diff
@@ -0,0 +1,29 @@
+class Foo extends ObservableObject {
+  static get props() {
+    return {
+      foo: "number"
+    };
+  }
+
+  static seal = true;
+}
+
+class MyTodo extends ObservableObject {
+  static get props() {
+    return {
+      foo: "number"
+    };
+  }
+
+  static seal = true;
```

Comment:
If the DefineMap had `seal: false`, we should just leave off the `static seal` property so the object will not be sealed.

Reply:
This is still wrong isn't it, @cherifGsoul?

Reply:
DefineMap are sealed, so if there's no explicit `sealed: false` the ObservableObject sealed!